input/fsh/instances/ror-serveur.fsh

```diff
@@ -1,38 +1,36 @@
 Instance: ror-serveur
```

Comment:
Pour les URL HL7, comme les paramètres de recherche de base FHIR sushi me retourne une erreur disant qu'il ne connait pas l'URL. Est ce que je dois ajouter fhir.r4 dans le dossier de dépendance ?

Reply:
Ca m'étonne un peu, tu as essayé avec http://hl7.org/fhir/SearchParameter/HealthcareService-organization et Canonical(HealthcareService-organization) et dans les deux cas ça n'a pas fonctionné ?

Pour moi ça ne devrait pas être nécessaire, mais tu peux essayer :)

Reply:
Les CS de MHD sont faits en json, il y a peut être un pb à ce sujet : https://github.com/IHE/ITI.MHD/tree/master/input/resources

Reply:
Initialement on avait : * rest.resource[=].searchParam[=].definition = "http://hl7.org/fhir/SearchParameter/HealthcareService-organization" et ça fonctionnait, j'ai essayé avec Canonical(HealthcareService-organization) et j'ai cette erreur : 
error Cannot use canonical URL of HealthcareService-organization because it does not exist. Be sure that HealthcareService-organization exists and it has a URL.
  File: C:\Users\tru\Documents\GitHub\ROR_ressource-conformance\input\fsh\instances\ror-consommateur.fsh

Reply:
même avec l'ajout du tgz du package dans les dépendances on garde l'erreur !

Reply:
Ca marche, dommage!

Du coup je vois pas d'inconvénient à laisser l'url entière :)
C'est bien d'utiliser Canonical() pour les ressources définies dans la spec et éviter de tout remodifier, pour les RC HL7 c'est très stable donc pas de problème pour mettre l'url à mon sens :)